.gitignore

```diff
@@ -1 +1,5 @@
-Diagram/_other.cs
\ No newline at end of file
+Diagram/_other.cs
```

Comment:
Please revert

Reply:
Okay, I just noticed it wanted to add all build artifacts to git, so I figured the .gitignore should be updated. I'll revert this for the Pullrequest.